ethapi/block_cert_json.go

```diff
@@ -8,12 +8,12 @@ import (
 	"github.com/ethereum/go-ethereum/common"
 )
 
-// blockCertificateJson is a JSON representation of a block certificate
+// BlockCertificateJson is a JSON representation of a block certificate
```

Comment:
Since you are already touching these, maybe we should also drop the `Json` suffix. This is quite silly, and the original motivation was to avoid name collisions. But in this package there should be none, and being in the `ethapi` package it should be pretty clear that this is the format used for exporting certificates through the RPC API. Wdyt?